src/uu/dmesg/src/dmesg.rs

```diff
@@ -30,6 +38,12 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     let matches: clap::ArgMatches = uu_app().try_get_matches_from(args)?;
     if let Some(kmsg_file) = matches.get_one::<String>(options::KMSG_FILE) {
         dmesg.kmsg_file = kmsg_file;
+        dmesg.kmsg_record_separator = 0;
+    } else if cfg!(target_os = "windows") {
+        return Err(USimpleError::new(
+            1,
+            "Windows OS requires the use of '-K' switch",
```

Comment:
I think you can simplify the error message to something like
```suggestion
            "Windows requires the use of '-K'",
```

Reply:
I did the change, however I am not a native English speaker .